code/datums/helper_datums/map_template.dm

```diff
@@ -64,4 +65,50 @@
 	var/list/filelist = flist(path)
 	for(var/map in filelist)
 		var/datum/map_template/T = new(path = "[path][map]", rename = "[map]")
-		map_templates[T.name] = T
\ No newline at end of file
+		map_templates[T.name] = T
+
+	if(!config.disable_space_ruins) // so we don't unnecessarily clutter start-up
+		preloadRuinTemplates()
+	//preloadShuttleTemplates()
+
+/proc/preloadRuinTemplates()
+	// Still supporting bans by filename
+	var/list/banned
+	if(fexists("config/spaceRuinBlacklist.txt"))
+		banned = generateMapList("config/spaceRuinBlacklist.txt")
+	else
+		banned = generateMapList("config/example/spaceRuinBlacklist.txt")
```

Comment:
if it hasn't been copied over this should probably just fail gracefully, not go "oh well take example"